descriptors/ModuleDescriptor-template.json

```diff
@@ -18,8 +18,13 @@
         },
         {
           "methods": [ "POST" ],
-          "pathPattern": "/refresh",
```

Comment:
Is it intended that this API is removed at this time?

Reply:
I think yes this API can be removed because it was never used in code and removing it means one less thing for us to maintain. On the other hand, the `/refreshtoken` route (to get a signed refresh token but still - "legacy old code") is currently called by mod-login, although the token it gets is not used, it is still called, so I left it in so as to not break mod-login.

Originally I had also removed `/refreshtoken` as well but @adamdickmeiss  advised keeping it in so as to not be a breaking change and I agreed since I'd have to make changes to mod-login before merging this if I took it out. So even though this removes the `/refresh` route it can still be merged.

Reply:
In theory it's a breaking change, in practice nobody notices.